lib/routes/uic/career.ts

```diff
@@ -0,0 +1,70 @@
+import { Route } from '@/types';
+import puppeteer from '@/utils/puppeteer';
+import * as cheerio from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/career/:tag',
+    categories: ['university'],
+    example: '/uic/career/ygtz',
+    parameters: {
+        tag: 'tag, ygtz, xsfx, gg, xngs, msjq, sjzxx',
+    },
+    features: {},
+    name: 'UIC Career',
+    maintainers: ['heimoshuiyu'],
+    handler,
+};
+
+async function handler(ctx) {
+    const tag = ctx.req.param('tag');
+    const rootUrl = 'https://career.uic.edu.cn/news/index/tag/' + tag;
+
+    const browser = await puppeteer();
+    const page = await browser.newPage();
+    await page.setRequestInterception(true);
+    page.on('request', (request) => {
+        if (['image', 'stylesheet', 'font'].includes(request.resourceType())) {
+            request.abort();
+        } else {
+            request.continue();
+        }
+    });
+    await page.goto(rootUrl, {
+        waitUntil: 'networkidle2',
+    });
+    const response = await page.content();
+    await page.close();
```

Comment:
No need to use puppetter.
You can extract the compressed content using regex
 https://github.com/DIYgod/RSSHub/blob/379aa7d98f95820e9498b8671bac865df0b1788c/lib/routes/csu/career.ts#L56-L62